modular_skyrat/modules/blueshield/code/modules/jobs/job_types/blueshield.dm

```diff
@@ -29,12 +29,12 @@
 	uniform = /obj/item/clothing/under/rank/security/blueshield
 	id = /obj/item/card/id/silver
 	suit = /obj/item/clothing/suit/armor/vest/blueshield
-	suit_store = /obj/item/gun/ballistic/automatic/pistol/m1911
-	gloves = /obj/item/clothing/gloves/krav_maga/combatglovesplus
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/blueshield
+	gloves = /obj/item/clothing/gloves/tackler/combat/insulated
 	shoes = /obj/item/clothing/shoes/jackboots
 	ears = /obj/item/radio/headset/heads/blueshield/alt
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
-	backpack_contents = list(/obj/item/ammo_box/magazine/m45 = 3,/obj/item/melee/baton/blueshieldprod = 1)
+	backpack_contents = list(/obj/item/ammo_box/magazine/multi_sprite/ladon/blueshield/b460 = 3,/obj/item/melee/baton/blueshieldprod = 1)
```

Comment:
Haha you know what I'm gonna say here don't you? Make it a gunkit :3

Reply:
Gunkits are for armadyne stuff :^(

Reply:
but alas, sure